vue/src/lib/SignalRAspNetCoreHelper.ts

```diff
@@ -1,15 +1,15 @@
 import AppConsts from './appconst'
 import Util from './util'
-class SignalRAspNetCoreHelper{
-    initSignalR(){
+class SignalRAspNetCoreHelper {
+    initSignalR() {
         var encryptedAuthToken = Util.abp.utils.getCookieValue(AppConsts.authorization.encrptedAuthTokenName);
-
```

Comment:
Let's restore this empty line for consistency.

Reply:
You may have missed this.

Reply:
done, i restored this empty line

Reply:
Please remove the trailing spaces. You can see that there is still undesirable diff.